_docs/content/basics/06_reactions.md

```diff
@@ -14,7 +14,7 @@ Currently the following chat adapters support reactions:
 
 - <i class="fas fa-terminal"></i> CLI Adapter: https://github.com/go-joe/joe
 - <i class='fab fa-slack fa-fw'></i> Slack Adapter: https://github.com/go-joe/slack-adapter
-- <i class='fab fa-rocketchat fa-fw'></i> Rocket.Chat Adapter: https://github.com/dwmunster/rocket-adapter
+- <i class='fas fa-circle-notch'></i> Mattermost Adapter: https://github.com/dwmunster/joe-mattermost-adapter
```

Comment:
I didn't do much digging to confirm if this was the right class, just copy/pasted from one of the index pages.